ote/src/cljs/ote/app/controller/transit_visualization.cljs

```diff
@@ -102,11 +102,20 @@
 
         ;; Group by only-changes by route-hash-id
         grouped-changes (group-by :route-hash-id only-changes)
+        group-recent? (map
+                         (fn [[_ changes]]
+                           (some? (some #(:recent-change? %) changes)))
+                         grouped-changes)
         ;; Take first from every vector
         route-changes (map #(first (second %)) grouped-changes)
+        route-changes-with-recent (map
```

Comment:
Alternative approach would be to replace `route-changes-with-recent` with by adding into `group-recent?` following assoc
```
 route-changes (map 
  #(let [res (first (second %))]
      (assoc res :recent-change? val) ;; val = true if filter for second returns any items with  :recent-change?
  grouped-changes)
```